core/templates/pages/about-foundation-page/about-foundation-page.component.ts

```diff
@@ -16,29 +16,45 @@
  * @fileoverview Component for the about foundation page.
  */
 
-import { Component, OnInit } from '@angular/core';
+import { Component, OnInit, OnDestroy } from '@angular/core';
 import { downgradeComponent } from '@angular/upgrade/static';
+import { Subscription } from 'rxjs';
 
 import { PageTitleService } from 'services/page-title.service';
 import { UrlInterpolationService } from 'domain/utilities/url-interpolation.service';
+import { TranslateService } from '@ngx-translate/core';
```

Comment:
I would also move this together with the other library imports (`@angular/core`)

Reply:
ditto elsewhere

Reply:
Done!